src/fhir/api.ts

```diff
@@ -128,3 +128,43 @@ const createEntry = (resource: Resource, identifier?: string) => {
     request: requestInfo,
   };
 };
+
+export const checkResponseOK = async (response: Response) => {
```

Comment:
the checkResponseOK which throws an exception can probably be in test utils, then the functino which goes through the bundle response can be here